app/adapters/special-purpose-district.js

```diff
@@ -12,10 +12,10 @@ const SQL = function(id) {
 };
```

Comment:
Note that we should be using `dcp_special_purpose_districts` here.

Reply:
Yea I'll capture that in a bug or task. It also looks like `zoning-map-amendment.js` references `zoning_map_amendments` but a data set called `dcp_zoning_map_amendments` exists.